### ISSUE TITLE
Meta Knight on ledge Crash

### ISSUE THREAD

**Issue by MetaKnight107** [TheOriginalSINe/Project-M-CC#433]
So when I would play Project M cc I would be Meta Knight on turbo mode and Meta Knight would be on a ledge and someone would be near (and sometimes do a doge roll) and the game would crash it has been bugging me for a long time so pls fix it soon

**Comment by MetaKnight107** [comment]
Oh and could you explain why you are switching to PMEX? Would that just make it easier for the game to crash?

**Comment by bb010g** [comment]
It would make all of these issues not our problem, mainly. PMEX can be made more stable by more people who know more of what they're doing.

**Comment by TheSmashBoy** [comment]
@MetaKnight107 And also, it is much easier, better. Versatile and flexible. And also because bb010g and a few others had it with the shitstorm of what should be and what should not be on a future complete vPM 3.61-3.7 build and just decided to switch to PMBEX.

**Comment by MetaKnight107** [comment]
Is the PMBEX version almost complete? Or are you guys planning on releasing one more version that isn't BrawlEx?

**Comment by TheSmashBoy** [comment]
@MetaKnight107 No. bb010g probably won't make a vPMBEx+CC build since he will only provide the character files and people go learn how to add them themselves. That's the current RFC of the community.

**Comment by bb010g** [comment]
I've still got a desire to make some PMEX tooling, but it's not too high on my priority list. I'd mostly like to get to a state where a vanilla PMEX + Knuckles build could be automated first, as Knuckles seems to be the most popular of the newcomers and I'd like to finally transition away from our weird not quite 3.61 mishmash sooner rather than later.
